lib/load.php

```diff
@@ -86,11 +86,17 @@ function gutenberg_is_experiment_enabled( $name ) {
 require dirname( __FILE__ ) . '/compat.php';
 require dirname( __FILE__ ) . '/utils.php';
 
+// Include FSE related files only if the experiment is enabled.
+if ( gutenberg_is_experiment_enabled( 'full-site-editing' ) ) {
+	require dirname( __FILE__ ) . '/templates.php';
```

Comment:
Should we remove the checks in those files now then?

Reply:
that sounds good to me